devices/HapticGlove/CMakeLists.txt

```
@@ -0,0 +1,48 @@
+#/*
+# Copyright 2021 Istituto Italiano di Tecnologia (IIT)
+# @author Kourosh Darvish <kourosh.darvish@iit.it>
+#* Authors: Yeshasvi Tirupachuri
+#* CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+#*/
+
+find_package(Eigen3 REQUIRED)
+include_directories(SYSTEM ${EIGEN3_INCLUDE_DIR})
```

I think that the `NO_MODULE` version of Eigen3 does not require using `include_directories`, just link against `Eigen3::Eigen`.

```suggestion
find_package(Eigen3 REQUIRED NO_MODULE)
```